### PR TITLE
Lup teleport text

### LINKED ISSUE #69: World Builder League Rocket
When you travel to a world builder league world it says "Do you want to travel to Nexus Tower?"

### PR DESCRIPTION
**Pull request by aronwk-aaron**
Add logic to use zone summary if not going to NT or NS

Resolves #69 

Tested with all the lup worlds and with the lego club teleporter and everything behaved as expected

### CODE REVIEW

**Comment by aronwk-aaron** [comment]
Screenshot from 2022-05-02 21-08-55
Ex of what is displayed now

File: dScripts/ChooseYourDestinationNsToNt.cpp [hunk 1]
```diff
@@ -59,14 +59,11 @@ void ChooseYourDestinationNsToNt::BaseChoiceBoxRespond(Entity* self, Entity* sen
...
-        {
+        if (newMap == 1200) {
             strText = u"UI_TRAVEL_TO_NS";
-        }
-        else
-        {
+        } else if (newMap == 1900) {
             strText = u"UI_TRAVEL_TO_NEXUS_TOWER";
-        }
+        } else strText = u"ZoneTable_" + GeneralUtils::ASCIIToUTF16(std::to_string(newMap)) + u"_summary";
```
Comment (Wincent01) [inline dScripts/ChooseYourDestinationNsToNt.cpp:66]: There is a GeneralUtils::to_u16string() if I recall correctly.

**Review by Wincent01 — COMMENTED** [review]
A minor request, otherwise looks good

File: dScripts/ChooseYourDestinationNsToNt.cpp (hunk 1 above)
Comment (aronwk-aaron) [reply dScripts/ChooseYourDestinationNsToNt.cpp:66]: Quite literally a few lines below that to_u16string is used :upside_down_face: silly me
Thanks for pointing it out!

**Comment by aronwk-aaron** [comment]
Closing in favor of a proper fix

**Pull request closed without merging**